egs/aishell/v1/conf/mfcc.conf

```diff
@@ -0,0 +1,2 @@
+--use-energy=false   # only non-default option.
```

Comment:
This looks funny to me... Does it still make sense to use the energy VAD if --use-energy=false? @danpovey, could you comment on this one?